File: g3malt/src/main/java/isaapp/g3malt/dto/UpdateBloodBankStorageDto.java
package isaapp.g3malt.dto;

public class UpdateBloodBankStorageDto {
	
	private Integer bloodBankId;
	private String bloodType;
	
	public UpdateBloodBankStorageDto() {}
	
	public UpdateBloodBankStorageDto(Integer bloodBankId, String bloodType) {
		super();
		this.bloodBankId = bloodBankId;
		this.bloodType = bloodType;
	}

	public Integer getBloodBankId() {
		return bloodBankId;
	}

	public void setBloodBankId(Integer bloodBankId) {
		this.bloodBankId = bloodBankId;
	}

	public String getBloodType() {
		return bloodType;
	}

	public void setBloodType(String bloodType) {
		this.bloodType = bloodType;
	}
}

File: frontend/src/app/DTO/update-blood-bank-storage-dto.ts
export class UpdateBloodBankStorageDto {
    bloodBankId!: number;
    bloodType!: string;

    public constructor(id?: number, bloodType?: string) {
        if (id || bloodType) {
          this.bloodBankId = id!;
          this.bloodType = bloodType!;
        }
    }
}